docs/blog/newsletter-october-2017.rst

```diff
@@ -0,0 +1,81 @@
+
+In the meantime, check out #documenting-apis or #apithedocs for more conversations about APIs!
+
+******************************************
+Proofreading and copyediting your own work
+******************************************
+If you're a lone writer, you've probably been the unenviable position of not having an editor around to proofread, copyedit, or otherwise sanity-check your writing. It's a perennial dilemma, and one that surfaces regularly on Slack. This month produced a particularly good batch of suggestions and resources. To help you clean up your copy when your only editor is you, here's some tips and tricks from the Write the Docs hivemind:
+
+* Put your writing away and review it at least a day later.
+* Print it out on paper and read/edit in the new format.
+* Read each sentence independently. Try reversing the order of sentences, starting with the last one and working your way backward to the first. Or reverse the order of paragraphs.
+* Listen to it being read aloud, either by you or by a device. Text-to-Speech tools are widely available.
+* For typos, repeats, and especially homophones, highlight a single line at a time. If it’s hardcopy, use a ruler. This keeps you from reading to the next line and helps you notice the “the the” problems.
+* Make the text much larger, or change the font. Folks recommended these tricks for on-screen or printed page reviews.
+* Last but not least, check out `this blog post of 10 tips for proofreading your own work <https://writetodone.com/get-your-eagle-eye-on-10-tips-for-proofreading-your-own-work/>`_. The post repeats some of our group's suggestions, and adds more.
+
+************************************************
+Defining developer relations/evangelism/advocacy
+************************************************
+This month, someone asked a question which many of us have wondered about, at one point or another: "What does a developer advocate or evangelist actually do?" (You may have even wondered this _as_ a devleoper advocate/evangelist...)
+
+The ensuing conversation made it clear that whether you call it developer relations (DevRel), developer evangelism, or developer advocacy, it can mean a lot of different things. But some common themes emerged:
+
+* Folks in these roles are generally expected to be the "face" of their developer community. Their work involves making their product/language/framework/etc. more visible and more approachable.
+* Often they sit at an overlap point between engineering – since they're (ideally) experts in the tool they're representing – and sales, marketing, documentation, etc.
+* The line between "evangelist" and "advocate" is blurry, but often the evangelist side is about bringing the product out to the community while the advocate side focuses on bring the community's voice back to the product team.
```

Comment:
I would break this sentence up - it's a little clunky, if well put.

Reply:
totally – good catch